public/icons/github.svg

```diff
@@ -1,5 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24">
```

Comment:
This one wasn't really a problem since we also use `currentColor` but it seemed better to remove it for consistency.